src/pudl/transform/params/ferc1.py

```diff
@@ -247,6 +247,56 @@
 ##############################################################################
 # String categorizations
 ##############################################################################
+PLANT_FUNCTIONAL_CLASSIFICATION_CATEGORIES = {
```

Comment:
I pulled out several string categorizations that show up in various places multiple times, but I didn't apply them outside of my own table yet. What do you think about switching over to using them where they apply?

Reply:
I thought we might concatenate the special "other" categories with a `dict | dict` in the specific params.